### PR TITLE
forge snapshot: don't execute fuzz tests at all by default

### PR DESCRIPTION
**Pull request by transmissions11**
closes https://github.com/gakonst/foundry/issues/1136

this is a significant speedup for solmate, from ~5.1s -> <1s

### CODE REVIEW

File: cli/src/cmd/forge/test.rs [hunk 1]
```diff
@@ -160,100 +160,8 @@ impl TestArgs {
...
-                runner,
-                verbosity,
-                filter,
-                self.json,
-                self.allow_failure,
-                (self.gas_report, config.gas_reports),
-            )
-        }
+    fn run(self) -> eyre::Result<Self::Output> {
+        custom_run(self, true)
```
Comment (transmissions11) [inline cli/src/cmd/forge/test.rs:164]: maybe we add a flag like --ignore-fuzz-tests so ppl can set this in their tests if they like?

Comment (transmissions11) [reply cli/src/cmd/forge/test.rs:164]: i think maybe save for a follow up pr

Comment (mattsse) [reply cli/src/cmd/forge/test.rs:164]: yeah followup and the whole test run would benefit from another pass.

File: cli/src/cmd/forge/test.rs [hunk 2]
```diff
@@ -390,24 +298,123 @@ fn short_test_result(name: &str, result: &forge::TestResult) {
...
+            }
+    } else {
+        let TestArgs { filter, .. } = args;
+        test(
+            runner,
+            verbosity,
+            filter,
+            args.json,
+            args.allow_failure,
+            include_fuzz_tests,
```
Comment (mattsse) [inline cli/src/cmd/forge/test.rs:392]: that's the only thing that changed after moving this function, right?

**Review by mattsse — APPROVED** [review]
overall, this is an improvement.
will try to refactor the test-related code shortly.

File: cli/src/cmd/forge/test.rs (hunk 2 above)
Comment (transmissions11) [reply cli/src/cmd/forge/test.rs:392]: yup!

**Pull request merged**